src/transformers/modeling_utils.py

```diff
@@ -784,6 +784,10 @@ def from_pretrained(cls, pretrained_model_name_or_path, *model_args, **kwargs):
             use_cdn(:obj:`bool`, `optional`, defaults to :obj:`True`):
                 Whether or not to use Cloudfront (a Content Delivery Network, or CDN) when searching for the model on
                 our S3 (faster). Should be set to :obj:`False` for checkpoints larger than 20GB.
+            mirror(:obj:`str`, `optional`, defaults to :obj:`None`):
+                Mirror source to use for downloads. It can be a URL or a string selected from a preset mirror sources.
+                For Chinese users, you can simply set this option to `tuna` to use the mirror provided by Tsinghua
+                University.
```

Comment:
I wouldn't document this option too much (I think this is fine) given that:
- we'll lose logs
- this might be temporary if we find a way to make Cloudfront work reliably in China (see https://aws.amazon.com/blogs/networking-and-content-delivery/china-delivery-is-here-amazon-cloudfront-announces-new-cdn-in-mainland-china/ for instance)